### PR TITLE
Preview

### PR DESCRIPTION
**Pull request by dmorina**

### CODE REVIEW

**Comment by dmorina** [comment]
@mbernst check and adjust the language if necessary 
` 46 static/templates/project/preview.html `

**Comment by mbernst** [comment]
Will look! Need to reinstall dev environment on new laptop. Yayayayay

**Comment by shirishgoyal** [comment]
@dmorina can you check the fix I pushed for height issue? it should not require opening new page

**Comment by dmorina** [comment]
We got rid of it, now it opens a new tab

**Comment by mbernst** [comment]
We need to decide if preview is task level (a la Mechanical Turk) or project level.

I'm seeing some weird behavior. I accept the project with 494 tasks. Then I press "skip" to skip the first task, and it returns everything and takes me back to the task feed. Shouldn't it take me to the second task in the project? Actually, the same thing happens even if I submit the task. I think this has to do with auto-accept?

**Comment by dmorina** [comment]
correct, that's what auto accept does, if unchecked it will send you back to taskfeed

**Comment by dmorina** [comment]
what does a project level preview mean? is that just the template showing the {{variables}}?
I thought we decided to render first row of the csv ?

**Comment by mbernst** [comment]
Yeah it previews the project instead of each task. On AMT you preview each task before accepting, which is why autoaccept is necessary

**Comment by mbernst** [comment]
Need to think about the right model...

**Comment by dmorina** [comment]
Ok then, let me know

**Comment by dmorina** [comment]
However, 4 of the workers who have been working on your tasks have auto-accept enabled so it doesn't seem to be an issue as it is right now, also there were no complaints from them about this.
My vote would be just keep it as it is until someone faces an issue.

File: static/templates/project/preview.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        <span class="_nav-title _truncated-text" style="max-width: 650px">{{ taskfeed.previewedProject.name }}</span>
+
+        <div class="_content-actions">
+
+
+        </div>
+    </div>
+    <div class=" _error-inset" style="margin: 16px 16px 16px 32px"
+         ng-if="project.project.status==project.status.STATUS_IN_PROGRESS || project.project.status==project.status.STATUS_PAUSED">
+        You are previewing this project. Click "Accept" to start working on it.
```
Comment (dmorina) [inline static/templates/project/preview.html]: @mbernst do you have a sentence for me here?

Comment (mbernst) [inline static/templates/project/preview.html]: That's fine for now. I need to figure out the broader situation of previewing...

Comment (mbernst) [inline static/templates/project/preview.html]: In the meantime lgtm

**Pull request merged**